### ISSUE TITLE
Save a new sowing intervention fails

### ISSUE THREAD

**Issue by EmmaDorie** [ekylibre/ekylibre#518]
At the creation of a new sowing intervention, the procedure can't be save :
```
#<ActiveModel::Errors:0x007fac1f0bb300 @base=#<Intervention id: nil, issue_id: nil, prescription_id: nil, procedure_name: "sowing", state: "undone", started_at: nil, stopped_at: nil, created_at: nil, updated_at: nil, creator_id: nil, updater_id: nil, lock_version: 0, event_id: nil, number: "11", description: "", working_duration: 0, whole_duration: 0, actions: [:sowing], custom_fields: nil>, @messages={:"inputs.quantity_population"=>["Quantité population ne peut pas être vide"]}> 
```
"quantity_population" was completed and all the different units had been tested.
This issue is not reproduce systematically.

**Comment by EmmaDorie** [comment]
Same error with all planting interventions.

**Comment by burisu** [comment]
Fixed with b673d68e0f5c7566874d2a8e35d29006909926c9.
